src/main/java/com/_up/megastore/controllers/requests/RecoverPasswordRequest.java

```diff
@@ -0,0 +1,15 @@
+package com._up.megastore.controllers.requests;
+
+import jakarta.validation.constraints.NotNull;
+
+import java.util.UUID;
+
+public record RecoverPasswordRequest(
+        @NotNull(message = "Password must not be null")
+        String password,
+        @NotNull(message = "Confirm Password must not be null")
+        String confirmPassword,
```

Comment:
Los cambiaria a NotBlank para validar que no se ingrese una contraseña vacia

```suggestion
        @NotBlank(message = "Password must not be null")
        String password,
        @NotBlank(message = "Confirm Password must not be null")
        String confirmPassword,
```